ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1968,7 +1977,7 @@ KdbpCmdProc(
             State = ((Process->Pcb.State == ProcessInMemory) ? "In Memory" :
                     ((Process->Pcb.State == ProcessOutOfMemory) ? "Out of Memory" : "In Transition"));
 
-            KdbpPrint(" %s0x%08x  %-10s  %s%s\n",
+            KdbpPrint(" %s0x%p  %-10s  %s%s\n",
```

Comment:
Process IDs are DWORDs, they are just sometimes stored as HANDLEs. Printing out 64 bits is therefore unnecessary. Please change back to `%08x` and use `HandleToULong(Process->UniqueProcessId)`

Reply:
Done.